README.md

```diff
@@ -5,9 +5,9 @@
 
 ## Description
 
-Red Datasets provides classes that provide common datasets such as iris dataset.
+Red Datasets provides an easy way to fetch standard datasets commonly used in the data science community.
```

Comment:
I don't want to use "data science" here because we target more widely community.
We want normal Rubyists to use this library.

Reply:
Hmm.

Reply:
I'll think about it.

Reply:
Reverted. 
I think the idiom of the data science community is good. Because Ruby is also a language for data science and red-data-tools is a group that allows you to analyze data with Ruby. I think the data science community includes Ruby users. However, there is no big difference, so I will put it back.